terraform/aws/modules/route53-registration/variables.tf

```diff
@@ -1,19 +1,54 @@
-variable "deployment_name" {}
-
-variable "kubeconfig_dir" {}
-
-variable "private_hosted_zoneid" {}
-
-variable "public_hosted_zoneid" {}
-
-variable "enable_portal_public_r53_record" {}
-
-variable "enable_portal_private_r53_record" {}
-
-variable "enable_portal_internal_r53_record" {}
-
-variable "enable_awat_record" {}
-
-variable "enable_chimera_record" {}
-
-variable "enable_kubecost_record" {}
+variable "private_hosted_zoneid" {
+  type        = string
+  description = "The AWS private hosted zone ID"
+}
+
+variable "public_hosted_zoneid" {
+  type        = string
+  description = "The AWS public hosted zone ID"
+}
+
+variable "cloudflare_zone_id" {
+  type        = string
+  description = "The Cloudflare zone ID provided"
+}
+
+variable "enable_portal_public_r53_record" {
+  type        = bool
+  description = "Enables to create a public route53 record for public Customer Web Server"
+}
+
+variable "enabled_cloudflare_customer_web_server" {
+  type        = string
```

Comment:
is this a string or bool?

Reply:
if this changes to bool the if statement above also needs to change

Reply:
correct this is a bool. nice catch